subsys/bluetooth/controller/Kconfig.ll_sw_split

```diff
@@ -341,8 +352,8 @@ config BT_CTLR_XTAL_THRESHOLD
 
 config BT_CTLR_SCHED_ADVANCED
 	bool "Advanced scheduling"
-	depends on (BT_MAX_CONN != 0) && BT_CTLR_SCHED_ADVANCED_SUPPORT
-	default y if !(BT_PERIPHERAL && !BT_CENTRAL)
+	depends on BT_CTLR_SCHED_ADVANCED_SUPPORT && (BT_CONN || (BT_CTLR_ADV_EXT && (BT_CTLR_ADV_AUX_SET != 0) && (BT_CTLR_ADV_AUX_SET != 1)) || BT_CTLR_ADV_ISO)
```

Comment:
Should we worry about the maximum line length of 100 here?

Reply:
It can be done with `\`, e.g.

```
config NRFX_NVMC
	bool "NVMC driver"
	depends on $(dt_has_compat,$(DT_COMPAT_NORDIC_NRF51_FLASH_CONTROLLER)) \
		|| $(dt_has_compat,$(DT_COMPAT_NORDIC_NRF52_FLASH_CONTROLLER)) \
		|| $(dt_has_compat,$(DT_COMPAT_NORDIC_NRF53_FLASH_CONTROLLER)) \
		|| $(dt_has_compat,$(DT_COMPAT_NORDIC_NRF91_FLASH_CONTROLLER))
```